### DISCUSSION TITLE
Instance Method Error

### DISCUSSION THREAD

**Discussion by yawnicorn** [sst-inc/help#42] [Q&A - iOS] [answered]
<img width="1178" alt="Screenshot 2021-10-16 at 1 19 59 PM" src="https://user-images.githubusercontent.com/92565561/137574881-aa44b5f6-d7a7-4818-985f-559cd033cb92.png">
It shows an instance method error when I try to put an @IBAction for a button. How do I fix it?

**Comment by granwyntan** [comment, marked as answer]
It seems that your `@IBAction func UpgradeAutoClickerPressed() {}` is within the `override func viewDidLoad() {}` scope, so you can add a close bracket `}` on line 27 to close the `viewDidLoad()` override function. Hope it helps

**Reply by yawnicorn** [reply]
Ok Thank You
